src/System.Private.ServiceModel/src/System/IdentityModel/Claims/ClaimSet.cs

```diff
@@ -31,6 +32,26 @@ public static ClaimSet System
             }
         }
 
+        public static ClaimSet Windows
+        {
+            get
+            {
+#if FEATURE_NETNATIVE // NegotiateStream
+                throw ExceptionHelper.PlatformNotSupported("Windows Stream Security is not supported on UWP yet");
```

Comment:
Question: does compiling for NET Native give a warning about not using s_windows or assigning it a value? If so I wonder if its declaration should also be conditional on FEATURE_NETNATIVE.